File: lessons/python-beginner/08-清單/solutions/make.py
count = [0, 0, 0, 0, 0, 0]
n = int(input())
for i in range(n):
    v = int(input())
    count[v] = count[v] + 1
best = 1
for i in range(2, 6):
    if count[i] > count[best]:
        best = i
for i in range(1, 6):
    print(i, count[i])
print("當選", best)

File: lessons/cpp-beginner/11-for迴圈/solutions/make.cpp
int main() {
    int count = 0;
    for (int i = 1; i <= 100; i++) {
        if (i % 3 != 0) continue;
        if (i % 5 == 0) continue;
        count = count + 1;
    }
    cout << count << endl;
    return 0;
}
